tests/Unit/CssInlinerTest.php

```diff
@@ -2422,160 +2094,6 @@ function (array $dataset) {
         );
```

Comment:
These three self-closing tag data providers are no longer referenced and can be removed.

Reply:
Good catch! Done.